app/javascript/components/Facet.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import FiltersBox from './FiltersBox';
+
+/**
+ * Converts string value to lowercase, hyphen-delimited version
+ * e.g. "Cell type" -> "cell-type"
+ */
+function slug(value) {
+  return value.toLowerCase().replace(/ /g, '-');
+}
+
```

Comment:
I might rename this FacetControl, but maybe that's redundant

Reply:
Done in cd210e6.